Week3/LESSONPLAN.md

```diff
@@ -30,8 +30,14 @@ Create a repository and initialize GIT. Show the use of `git branch`, `git check
 
 ### Exercise
 
+A fun exercise by [Arco](https://github.com/ArcoMul) to practise creating pull requests: [Cat pull request exercise](https://github.com/ArcoMul/netlify-cats)
+
+Instruction on how to set things up at Netlify: https://github.com/ArcoMul/netlify-cats/blob/main/SETUP.md
+
 ### Essence
 
+_TODO_
```

Comment:
```suggestion
```

I would just leave this blank for now. This part rarely gets filled in and am going to see what the idea was behind it.